site/pages/create-disruption.page.tsx

```diff
@@ -65,6 +64,63 @@ const CreateDisruption = ({ inputs }: CreateDisruptionProps): ReactElement => {
         });
     };
 
+    const addValidity = (e: SyntheticEvent) => {
+        e.preventDefault();
+        setValidityClicked(true);
```

Comment:
The `validityClicked` is used to show errors when required and should be set to true when one or more of the validity inputs are empty

Reply:
done